Common/cpp/WorkletModule.cpp

```diff
@@ -129,4 +76,4 @@ void WorkletModule::setApplierId(int applierId) {
   this->applierId = applierId;
 }
 
-int WorkletModule::applierId = INT_MAX;
+int WorkletModule::Applier_Id = INT_MAX;
```

Comment:
Maybe we should move it to the ApplierRegistry? 
New_Local_Applier_Id?

Reply:
fixed 957dc94